pylintrc

```diff
@@ -17,7 +17,8 @@
 #W0622 - redefined-builti
 #C0325 - superfluous-parens
 #R0205 - useless-object-inheritance
-disable=C0103,R0904,R0903,W0511,W0232,R0922,R0801,R0921,W0141,R0401,I0013,W0142,W0622,C0325,R0205
+#W0614 - unused-wildcard-import
```

Comment:
Why add this here if they are disabled inline? It might mask other bad usage of wildcard imports in the future